app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,7 @@
     <string name="destination_location_label">Destination Location</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/ar-localizer-view-android/blob/6e7a43f049fda9487b1419ac862329af39e581fa/app/src/main/res/values/strings.xml#L5"></span>The resource <code>R.string.destination_location_label</code> appears to be unused</td>
    </tr>
  </tbody>
</table>